### PR TITLE
86b1vnx5r

### PR DESCRIPTION
**Pull request by charan2r**
I installed Jest and added the necessary dependencies for Babel and Typescript to work in the testing unit. After creating the test directory, I created a .ts test file and added sample tests to ensure they worked correctly. Then I created a .env.test environment file, added a database_url variable, and created a sample test to check whether it is working through process.env.<variable_name>

### CODE REVIEW

File: server/test/sum.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+function sum(a: number, b: number): number {
```
Comment (bsaranga) [inline server/test/sum.ts]: Remove this file. Not necessary for the project.

File: server/test/sum.test.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import sum from './sum';
```
Comment (bsaranga) [inline server/test/sum.test.ts]: Remove this also.

File: server/test/new.test.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { describe, test, expect } from '@jest/globals';
```
Comment (bsaranga) [inline server/test/new.test.ts]: Rename this file to `preliminaries.test.ts`

**Review by bsaranga — CHANGES_REQUESTED** [review]

File: server/test/new.test.ts (hunk 3 above)
Comment (charan2r) [reply server/test/new.test.ts]: File changes were done.

Comment (thlurte) [reply server/test/new.test.ts]: i am converting this pr to draft, we will modify the babel config and get back.

Comment (charan2r) [reply server/test/new.test.ts]: Ahmed and I resolved the errors by creating the necessary config files and installing the required dependencies.
Screenshot (152)

**Review by bsaranga — APPROVED** [review]
Looks good. I assume the nextjs SWC errors were resolved by this.

**Pull request merged**